tasks/elasticsearch-RedHat.yml

```diff
@@ -19,7 +19,7 @@
   when: es_use_repository
   register: redhat_elasticsearch_install_from_repo
   notify: restart elasticsearch
-  until: '"failed" not in redhat_elasticsearch_install_from_repo'
+  until: not redhat_elasticsearch_install_from_repo.failed
```

Comment:
Does this work for you? 

For me locally and the [jenkins job](https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/21/OS=centos-7,TEST_TYPE=standard,VERSION=6.x,label=swarm/console) fails consistently with

```
18:44:52        TASK [elasticsearch : RedHat - Install Elasticsearch] **************************
18:45:02        fatal: [localhost]: FAILED! => {"msg": "The conditional check 'not redhat_elasticsearch_install_from_repo.failed' failed. The error was: error while evaluating conditional (not redhat_elasticsearch_install_from_repo.failed): 'dict object' has no attribute 'failed'"}
``` 

It seems to be failing because the "failed" result doesn't exist until the task has finished. I just tried `until: redhat_elasticsearch_install_from_repo.rc == 1` and it seems to work as expected.



![image](https://user-images.githubusercontent.com/4550136/36488507-e9610496-1723-11e8-9809-1209c0c885ce.png)